### ISSUE TITLE
[bug][native] text in Button inside Form.Trigger is not centered

### ISSUE THREAD

**Issue by G-Ray** [tamagui/tamagui#1568]
```js
<Form.Trigger asChild>
  <Button>
    Save
  </Button>
</Form.Trigger>
```

will render correctly on web, with a centered text.
Meanwhile on native, the text is not properly aligned:
![image](https://github.com/tamagui/tamagui/assets/2981774/98e09525-c977-4c0d-a252-e285b5c035cf)

**Comment by steffeli** [comment]
As a workaround adding `textProps={{alignSelf: 'center'}}` as a `Button` property helped me out center the text vertically.

**Comment by pierpo** [comment]
It helps, but if you have an icon in your button then the fix won't work 😄

**Comment by natew** [comment]
So `asChild="except-style"` works for now, looking at a better fix.

**Comment by natew** [comment]
Fix 164662ec79a9160a4b30d2ae0a79398723b388e3 going out in v1.68.4
